chart/templates/rendered.yaml

```diff
@@ -247,6 +250,19 @@ spec:
               syncProvider:
                 nullable: true
                 properties:
+                  httpSyncConfiguration:
+                    description: HttpSyncConfiguration defines the desired configuration
+                      for a http sync
+                    nullable: true
+                    properties:
+                      bearerToken:
+                        type: string
+                      target:
+                        description: Target is the target url for flagd to poll
+                        type: string
+                    required:
+                    - target
+                    type: object
```

Comment:
I think there are changes in this PR (such as this) which are only appearing because you need to rebase on main.